mkdocs/docs/HPC/python.md

```diff
@@ -0,0 +1,164 @@
+# Python 
+
+This page provides information on how to use Python on the HPC systems.
+It shows how to load Python modules and Python packages, 
+and how to run Python scripts interactively and using job scripts.
+
+## Python modules
+
+The HPC systems provide a number of Python modules that you can load into your environment.
+To see a list of available Python modules, you can run:
+
+```bash
+$ module avail Python/
+```
+
+For example, to load Python 3.11, you can run:
```

Comment:
There's also `Python/3.11.5-GCCcore-13.2.0`, maybe use that (since why would you use 3.11.3 if 3.11.5 is there too).

For recent `Python` modules, there are also companion modules that provide a bunch of Python packages from PyPI, like `Python-bundle-PyPI/2023.10-GCCcore-13.2.0`, `SciPy-bundle/2023.11-gfbf-2023b`, `matplotlib/3.8.2-gfbf-2023b`, etc.

These are quite common, so it's worth name-dropping them and mentioning `module show` to inspect what's provided by them so people don't feel a need to install common packages like `matplotlib`, `numpy`, etc.

I would also mention `pip list` to run after loading a bunch of modules, so people can easily check what's readily available already.

Reply:
fix in [change python version from 3.11.3 to 3.11.5](https://github.com/hpcugent/vsc_user_docs/pull/689/commits/5b496b381ed62724737ac9b88d1da8a3afe77cf2)